sklearn/impute/_iterative.py

```diff
@@ -194,6 +197,16 @@ class IterativeImputer(_BaseImputer):
     Features which contain all missing values at :meth:`fit` are discarded upon
     :meth:`transform`.
 
+    Using defaults, the imputer scales in :math:`\\mathcal{O}(knp^3\\min(n,p))`
+    where :math:`k` = `max_iter`, :math:`n` the number of samples and
+    :math:`p` the number of features. It thus becomes prohibitively costly when
+    the number of features increases. Setting
+    `n_nearest_features << n_features`, `skip_complete=True` or increasing `tol`
```

Comment:
I noticed that
```
`n_nearest_features` << `n_features`, `skip_complete` = `True`
```
was changed to 
```
`n_nearest_features << n_features`, `skip_complete=True`
```
in [`f5a1ebd` (#23701)](https://github.com/scikit-learn/scikit-learn/pull/23701/commits/f5a1ebd44ec7dc6b73f044cfc5cdb9c701153796). I agree that `skip_complete=True` is the correct markdown as the `=` operator is part of the user syntax, but I am wondering if the notation `n_nearest_features` << `n_features` is more correct for such a mathematical (and not user) syntax. What do you think?

Reply:
I think that this is fine. We are not super consistent in this regard. At least, the HTML rendering should be on the same line then.